packages/shortcuts-extension/src/components/ShortcutList.tsx

```diff
@@ -23,10 +25,86 @@ export interface IShortcutListProps {
   height: number;
   contextMenu: Function;
   external: IShortcutUIexternal;
+  id?: string;
```

Comment:
I would consider renaming this to "index" to avoid confusion with DOM element's `id` property.

Reply:
Oh, no this is actually used as an ID. Please disregard.